src/components/Contentful/DynamicHero.vue

```diff
@@ -173,6 +173,24 @@ export default {
 				link: this.heroText?.primaryCtaLink ?? '',
 			};
 		},
+		heroBackground() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('background') > -1);
+		},
+		heroBackgroundStyle() {
+			// Creates an object with conditional keys if heroBackground contains those elements
+			const styleObject = {
+				...(this.heroBackground?.backgroundColor && {
+					'background-color': this.heroBackground?.backgroundColor
+				}),
+				...(this.heroBackground?.backgroundImage?.file?.url && {
+					'background-image': `url("${this.heroBackground?.backgroundImage?.file?.url}")`
+				}),
+				...(this.heroBackground?.backgroundImage?.file?.url && {
+					'background-size': 'cover'
```

Comment:
I think this is the default we want. It will span the background full width.

Reply:
I think this is good for the css background image initial version